src/frontend/src/session/cursor_manager.rs

```diff
@@ -471,16 +487,31 @@ impl SubscriptionCursor {
                     ans.push(row);
                 }
                 None => {
```

Comment:
Does `None` here always mean no data? I saw that we will return None when field mismatches [here](https://github.com/risingwavelabs/risingwave/blob/c47a2e8fce4a1a5bbd4f422b816be2b18460215f/src/frontend/src/session/cursor_manager.rs#L368).

Reply:
Yes, none means that there is no data for this call to next_row